app/components/chart/header.js

```diff
@@ -39,28 +39,33 @@ var TidelineHeader = React.createClass({
   },
   render: function() {
     var dayLinkClass = cx({
+      'js-day': true,
       'patient-data-subnav-active': this.props.chartType === 'daily',
       'patient-data-subnav-hidden': this.props.chartType === 'no-data'
     });
 
     var modalLinkClass = cx({
+      'js-modal': true,
       'patient-data-subnav-active': this.props.chartType === 'modal',
       'patient-data-subnav-hidden': this.props.chartType === 'no-data'
     });
 
     var weekLinkClass = cx({
+      'js-week': true,
       'patient-data-subnav-active': this.props.chartType === 'weekly',
       'patient-data-subnav-hidden': this.props.chartType === 'no-data'
     });
 
     var dateLinkClass = cx({
+      'js-date': true,
       'patient-data-subnav-text' : this.props.chartType === 'daily' || this.props.chartType === 'weekly' || this.props.chartType === 'modal',
       'patient-data-subnav-dates-daily': this.props.chartType === 'daily',
```

Comment:
would it make sense to have `'daily'` as a const? Same with other strings for weekly etc